frontend/src/Frontend/UI/DeploymentSettings.hs

```diff
@@ -695,3 +723,23 @@ signingItem (n, _) = do
 toggleCheckbox :: Reflex t => Dynamic t Bool -> Event t a -> CheckboxConfig t
 toggleCheckbox val =
   (\v -> def { _checkboxConfig_setValue = v }) . fmap not . tag (current val)
+
+-- parsed: "{\"role\": \"GAS\", \"description\": \"Pay the GAS required for this transaction\", \"cap\": {\"args\": [\"doug\",], \"name\": \"coin.GAS\"}}"
+defaultGASCapability :: DappCap
+defaultGASCapability = DappCap
+  { _dappCap_role = "GAS"
+  , _dappCap_description = "Pay the GAS required for this transaction"
+  , _dappCap_cap = PC.SigCapability
+    { PC._scName = PN.QualifiedName
+      { PN._qnQual = PN.ModuleName
+        { PN._mnName = "coin"
+        , PN._mnNamespace = Nothing
+        }
+      , PN._qnName = "GAS"
```

Comment:
The capability for gas is `coin.FUND_TX`

Reply:
Ta, fixing